lib/command/src/Obelisk/Command/Run.hs

```diff
@@ -386,9 +387,6 @@ parsePackagesOrFail dirs' = do
       "No valid, buildable packages found" <> (if null dirs then "" else " in " <> intercalate ", " dirs)
     Just xs -> pure xs
 
-  unless (null pkgDirErrs) $
-    putLog Warning $ T.pack $ "Failed to find buildable packages in " <> intercalate ", " pkgDirErrs
```

Comment:
Is this change related?

Reply:
It's related in that you also get this warning for a package with no library component, in addition to the error above. I don't think either of them should be considered out of the ordinary.

Reply:
Ah and there is no more `Either`.